src/components/modal/report/modal.jsx

```diff
@@ -101,21 +102,21 @@ class ReportModal extends React.Component {
                         </div>
                     </div>
 
-                    <div className="report-modal-content">
-                        <FormattedMessage
-                            id={`report.${type}Instructions`}
-                            values={{
-                                CommunityGuidelinesLink: (
-                                    <a href="/community_guidelines">
-                                        <FormattedMessage id="report.CommunityGuidelinesLinkText" />
-                                    </a>
-                                )
-                            }}
-                        />
-                        <Form
-                            className="report"
-                            onSubmit={onReport}
-                        >
+                    <Form
```

Comment:
The changes here were from before the project reporting work; they're here because I saw that what I was doing with the base modal and addtostudio scss needed to be reflected here too. I'd prefer to leave them in, if that's ok?

Reply:
👍